src/list/list.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Post, Body, Param, Put, Delete, UseGuards, SetMetadata } from '@nestjs/common';
+import { ListService } from './list.service';
+import { Create, Updata, List } from './dto/list.dto';
+import { AuthGuard } from '../auth/auth.guard';
+import { roles } from '../auth/auth.constants';
+import { Roles } from '../roles.decorator';
+
+@Controller('list')
+@Roles(roles)
```

Comment:
wildcard is not representative for studing project
please make some difference in access level for admin and user